src/main/java/dk/kb/util/BuildInfoManager.java

```diff
@@ -97,7 +193,7 @@ private static synchronized void loadBuildInfo() {
         try (InputStream is = Resolver.resolveStream(BUILD_PROPERTY_FILE)) {
             if (is == null) {
                 log.warn("Unable to load '" + BUILD_PROPERTY_FILE + "' from the classpath. " +
-                         "Build information will be unavailable");
+                        "Build information will be unavailable");
             } else {
                 properties.load(is);
```

Comment:
Please add a debug line stating that properties were loaded (and where they were loaded from).